packages/firestore/src/core/firestore_client.ts

```diff
@@ -135,13 +137,14 @@ export class FirestoreClient {
    * fallback succeeds we signal success to the async queue even though the
    * start() itself signals failure.
    *
-   * @param usePersistence Whether or not to attempt to enable persistence.
+   * @param persistenceSettings Settings object for the suggested persistence
+   *     mode.
```

Comment:
"Settings object for the suggested persistence mode" sounds weird to me.  Maybe "Whether persistence is enabled and any associated settings" or use the same description as you used below ("Settings object to configure offline persistence")

Reply:
I was trying to indicate that this is more of Christmas-type situation and that not every wish is going to be fulfilled. But it does sound a little weird. Updated to use "Settings object to configure offline persistence".